docs/version-specific/supported-software.md

```diff
@@ -27577,8 +27582,6 @@ version   |versionsuffix                |toolchain
 ``1.12.1``|                             |``foss/2021a``, ``foss/2022a``
 ``1.12.1``|``-CUDA-11.3.1``             |``foss/2021a``
 ``1.12.1``|``-CUDA-11.7.0``             |``foss/2022a``
-``1.13.1``|                             |``foss/2022a``
-``1.13.1``|``-CUDA-11.7.0``             |``foss/2022a``
```

Comment:
Not sure how this got in here, probably local WIP easyconfigs - we don't have PyTorch 1.13.1 yet (we don't even have PRs for it yet)...

Reply:
Sorry. This happened when I was battling generating these pages. I was struggling to sort out a working system to generate this page on (see my comment in #90 about the number of changes I was getting at one point).

Reply:
Ah, that clears it up. No worries :)